benchmark/opperf/README.md

```diff
@@ -47,7 +47,10 @@ Hence, in this utility, we will build the functionality to allow users and devel
 
 ## Prerequisites
 
-Make sure to build the flavor of MXNet, for example - with/without MKL, with CUDA 9 or 10.1 etc., on which you would like to measure operator performance. Finally, you need to add path to your cloned MXNet repository to the PYTHONPATH.
+Provided you have MXNet installed (any version >= 1.5.1), all you need to use opperf utility is to add path to your cloned MXNet repository to the PYTHONPATH.
```

Comment:
It would be great to give a one line example.

Reply:
In the note I have added link to how to install MXNet (page) instead of giving just 1 of the 10 different ways to install mxnet